client/src/modules/inventory/list/modals/actions.tmpl.html

```diff
@@ -153,14 +153,22 @@
       </div>
     </div>
 
-
     <div class="checkbox">
       <label>
         <input type="checkbox" name="consumable" ng-true-value="1" ng-false-value="0" ng-model="$ctrl.item.consumable">
         <span translate>FORM.LABELS.CONSUMABLE</span>
       </label>
     </div>
 
+    <div ng-if="$ctrl.item.consumable" class="checkbox">
```

Comment:
Is it possible for "non consumable" items to be packaged too?

Reply:
I don't think this notion could apply to services and other medical acts